widgets/HTTP-batchsource.json

```diff
@@ -684,27 +696,26 @@
       },
       "show": [
         {
-          "name": "authUrl",
-          "type": "property"
-        },
-        {
-          "name": "tokenUrl",
-          "type": "property"
+          "widget-type": "textbox",
+          "label": "Token URL",
```

Comment:
label is not required here.

Reply:
removed the labels attribute according to the filters conditions.